tests/jest-output.json

```diff
@@ -1,54 +1,154 @@
 {
-  "testResults": [{
-    "assertionResults": [{
-      "ancestorTitles": [
-        "my first describe"
-      ],
-      "fullName": "my first describe my first test",
-      "title": "my first test",
-      "status": "passed"
-    }, {
-      "ancestorTitles": [
-        "my first describe"
-      ],
-      "fullName": "my first describe my second test",
-      "title": "my second test",
-      "status": "passed"
-    }],
-    "status": "passed"
-  }, {
-    "assertionResults": [{
-      "ancestorTitles": [
-        "my second describe"
-      ],
-      "fullName": "my second describe my first test",
-      "title": "my first test",
-      "status": "failed"
-    }, {
-      "ancestorTitles": [
-        "my first describe"
-      ],
-      "fullName": "my second describe my second test",
-      "title": "my second test",
-      "status": "passed"
-    }],
-    "status": "failed"
-  }, {
-    "assertionResults": [{
-      "ancestorTitles": [
-        "my third describe"
-      ],
-      "fullName": "my third describe my first test",
-      "title": "my first test",
-      "status": "failed"
-    }, {
-      "ancestorTitles": [
-        "my first describe"
-      ],
-      "fullName": "my third describe my second test",
-      "title": "my second test",
-      "status": "failed"
-    }],
-    "status": "failed"
-  }]
+  "numFailedTestSuites":1,
```

Comment:
Are the changes in this file intentional?

Reply:
Yes, I update the file to keep entire jest output.